### ISSUE TITLE
Discord RPC stuck on "Browsing Tidal"

### ISSUE THREAD

**Issue by HurleybirdJr** [Mastermindzh/tidal-hifi#449]
## Problem:
As title suggests, no matter what I do (reinstall, re-login, toggle RPC on and off, etc...) the rich presence never shows the current song playing and appears to only show "Browsing Tidal". I've also tried viewing the activity via the profile view, and that also shows the same thing.

![image](https://github.com/user-attachments/assets/38ecc45e-4eef-4155-be87-e1a424f9cfe1)

And with the settings for Discord RPC, I've tried toggling all in different configurations and renaming the texts, but nothing gets it out of the idle state. I've also tried changing songs since that used to help it update, but that doesn't work either.

![image](https://github.com/user-attachments/assets/93837b73-5e4e-480b-ba60-2f9b7a3431d9)

### Tidal-hifi version: 
5.15.0

### Discord version: 
stable 314046 (408a13e) Host 0.0.62 x64 Build Override: N/A Linux 64-bit (6.10.2-arch1-1)

**Comment by HurleybirdJr** [comment]
Having looked through other issues for possible fixes/workarounds, I tried running Tidal-hifi via the terminal and noticed that when I start playing a song with Discord RPC enabled, this error block keeps appearing every second or so:

```js
(node:11465) UnhandledPromiseRejectionWarning: Error: child "activity" fails because [child "assets" fails because [child "large_text" fails because ["large_text" length must be at least 2 characters long]]]
    at RPCClient._onRpcMessage (/opt/tidal-hifi/resources/app.asar/node_modules/discord-rpc/src/client.js:178:19)
    at IPCTransport.emit (node:events:517:28)
    at /opt/tidal-hifi/resources/app.asar/node_modules/discord-rpc/src/transports/ipc.js:138:18
    at decode (/opt/tidal-hifi/resources/app.asar/node_modules/discord-rpc/src/transports/ipc.js:92:5)
    at Socket.<anonymous> (/opt/tidal-hifi/resources/app.asar/node_modules/discord-rpc/src/transports/ipc.js:120:7)
    at Socket.emit (node:events:517:28)
    at emitReadable_ (node:internal/streams/readable:601:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:81:21)
(node:11465) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 30)
```

**Comment by HurleybirdJr** [comment]
## OK, so I just figured out the cause of the problem and a solution.

#### Turns out the album name was actually less than 2 chars:​
![image](https://github.com/user-attachments/assets/42d9693f-10f1-4e54-b256-b3c45d6dba9a)

This caused the API to freak out since it doesn't like largeImageTexts less than 2 chars, so I've added a fix in a PR #450 to mitigate this issue by adding an extra space, which Discord automatically trims as trailing white-space, so the API is happy and isn't visually borked either :)
